en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+An incoming event or message to Choreo or an outgoing API call from Choreo is a minimum of 1 step.
+
+
+## Pricing examples
+
+### Example 1: REST API 
+
+An organization has a flight booking system on Choreo. This system involves several APIs and databases. The scenario and the APIs being used in the system are described below.
+
+#### Scenario
+
+To make a booking through the web application, a client first searches for available flights and then proceeds to make the booking. The client application (web application) upon receiving the request makes a call to the [Search API](#searchAPI) to search for available flights. When the client receives the search results the client proceeds to make a reservation using the [Reservations API](#reservationsAPI).
+
+The following two diagrams illustrate the interactions in the system while searching and reserving a flight.
+<br/>
+
+#### Glossary
+- <a class="pricing" id="searchAPI"> Search API</a>: An external-facing API in Choreo that searches for the availability of flights and seats given the destinations.
+- <a class="pricing" id="bookingsAPI">Bookings API</a>: An internal API in Choreo that updates the Bookings database when a booking for a certain flight is made.
+- <a class="pricing" id="reservationsAPI">Reservations API</a>: An external-facing API in Choreo that is used by client applications (web apps, mobile apps) to make flight bookings.
+- <a class="pricing" id="twillioAPI">Twilio API</a>: A third-party API that is used by the Reservations API in Choreo to send SMS notifications to customers when a booking has been made, canceled, or updated.
+- <a class="pricing" id="executionTime">Execution time</a>: The execution time taken for a process is the time taken between  sending a request and recieving a response.
+<br/>
+<br/>
+
+#### Searching for a flight
+
+![Search for a flight](../assets/img/references/rest-api-example1.png)
+
+Let’s see how steps are calculated for the search process: 
+
+- As illustrated above, the client initiates a search for available flights through the web application. 
+- The web application then makes **1 API call to the [Search API](#searchAPI)** in Choreo. This is counted as **1 step**.
+- The search API in Choreo then makes **1 call to the Flights Database** to retrieve the information according to the search criteria entered by the client. This is counted as **1 step**. 
+
+- **Assuming the entire [execution time](#executionTime) for the search process takes less than 500ms** (starting from making a call to Choreo, and Choreo making a call to the database), the search process thereby results in a **total of 2 steps**.
+
+<br/>
+
+#### Reserving a flight
+
+![Reserving a flight](../assets/img/references/rest-api-example2.png)
+
+Let’s see how steps are calculated for the reservation process:
+
+- As illustrated above, the client initiates the reservation process through the Web Application.
```

Comment:
Are we using a specific web application here? "web application" mentioned in simple case easrlier in this page.

Reply:
This is referring to the Web Application in the diagram.